includes/class-sensei-admin.php

```diff
@@ -130,6 +131,22 @@ public function add_course_order() {
 		);
 	}
 
+	/**
+	 * Add Lesson order page to admin panel.
+	 *
+	 * @since  4.0.0
```

Comment:
We should mark this `@access private` so that other devs don't try to use it (since we're only making it public out of necessity for the hook to work).

Reply:
fix: https://github.com/Automattic/sensei/pull/4687/commits/b6e5dcbc1a59a9b20f955b6c360d0ec968e56377#diff-d8fb37cfbd6e262938ad7ebef55bb73e27759fb0b6e40d412985f5c7976243c5R174-R179